### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by AdamTheWizard**
edited styling of page and added brief instruction paragraph at the top of the page. I am clicking "Create pull request" as I am sure it is what I need to do but massive apologies if you have to reverse my changes through any kind of issue!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,37 +5,122 @@
...
+        <p>Big Logo:</p><input type="text" id="logo-big" class="update" value />
       </div>
       <div class="variable">
-        Logo in Medal:<input type="text" id="logo-small" class="update" value />
+         <p>Logo in Medal:</p><input type="text" id="logo-small" class="update" value />
       </div>
     </div>
+    
     <div class="result">
-      <a href="#" id="save-link" download="settings.html">Save Settings</a><br />
```
Comment (niccokunzmann) [inline index.html:34]: This was the only thing missing.
It works with styling!

**Comment by AdamTheWizard** [comment]
So I did it?! Apart from that last little bit?!

**Comment by niccokunzmann** [comment]
Yes.

**Comment by AdamTheWizard** [comment]
Ah Nicco man! Thank you so much for your help! My first ever pull request! Very pleased with that and although I would never have given up, I was very ready to take a break for a long time aha! If you dont mind, I'll follow you on here and help out wherever I can. Thanks again, really does mean a lot.

**Comment by niccokunzmann** [comment]
You can contribute as you like :) I like this talk about contributing to ZeroMQ https://www.youtube.com/watch?v=uj-li0LO_2g
I guess, Hintjens said something about "The kick, people get, when it is merged."^^

What would you like to work on? What is your goal?

**Comment by AdamTheWizard** [comment]
I'm downloading the desktop github now so it should be less painful next time. Unltimately I want to build software that helps people deal with mental health disorders. Currently though I am just happy to contribute to anything and everything with the goal of improving my knowledge of developing and github as a whole.

I'll watch that video as soon as I get a chance. I just find the whole githib website really confusing at the moment. Lots of different buttons with strange names and I've been very nervous to touch other peoples code.

This has given me a great boost though, did you add the Save as link back in or would you like me to? :)



Sent from my Samsung Galaxy smartphone.


-------- Original message --------

**Comment by niccokunzmann** [comment]
I added the save link back.

You can totally touch other peoples code. You can not directly change it
unless they invite you to.
There are some tutorials for git:
https://github.com/fossasia/flappy-svg/tree/master/tasks/learn_git

**Comment by AdamTheWizard** [comment]
Okay cool. I did think that was the case i itially but through the actual process, I grew anxious! Thanks for clearing that up, I'll make a start wherever I can but I may be requesting more help in the near future! Aha

Thanks for everything Nicco,

Adam :)



Sent from my Samsung Galaxy smartphone.


-------- Original message --------

**Pull request merged**